packages/common/tests/regex.test.ts

```diff
@@ -61,14 +60,23 @@ describe('regex.test', () => {
         'question?mark',
         'asterisk*char',
         'control\0char',
-        'another\ncontrol'
+        'another\ncontrol',
+        'file name', // (a space and an underscore)
+        '< tag >', // (spaces and less-than < and greater-than > characters)
+        'key : value', // (a space and a colon :)
+        'quote " example', // (a space and a double quote ")
+        'path / to / file', // (spaces and forward slashes /)
+        'C:\\ path \\ to \\ file', // (spaces and backslashes \)
+        'pipe | character', // (a space and a pipe |)
+        'question ? mark', // (a space and a question mark ?)
+        'star * char' // (a space and an asterisk *)
```

Comment:
your strings are good and self-explanatory. we should remove comments where they can be understood

Reply:
does it matter? ill remove them nonetheless

Reply:
done